Library/Homebrew/dev-cmd/create.rb

```diff
@@ -140,15 +140,12 @@ def create_cask(args:)
   end
 
   def create_formula(args:)
-    fc = FormulaCreator.new(args)
-    fc.name = if args.set_name.blank?
+    fc = FormulaCreator.new(args.set_name, args.set_version, !args.no_fetch?, args.HEAD?)
```

Comment:
Keyword arguments might be a nicer fit here, particularly if some more are added.

Reply:
I tried to do it the Python way `version=args.set_version` but `brew style` complained. I'll try the Ruby way with `version: args.set_version`.